### PR TITLE
Implement random generation in EO

### PR DESCRIPTION
**Pull request by mximp**
Implementation for #791

### CODE REVIEW

File: eo-runtime/src/test/java/org/eolang/PhiTest.java [hunk 1]
```diff
@@ -73,7 +76,8 @@ public void takesStandardPackage() {
     public void takesDirectly() {
         MatcherAssert.assertThat(
             new Dataized(
-                Phi.Φ.attr("org").get().attr("eolang").get().attr("math").get().attr("random").get()
+                Phi.Φ.attr("org").get().attr("eolang").get().attr("math")
+                        .get().attr("random").get().attr("pseudo").get()
```
Comment (yegor256) [inline eo-runtime/src/test/java/org/eolang/PhiTest.java:80]: @mximp configure your IDE to always use 4 spaces for all indents. In this module qulice is turned off, that's why this line doesn't break the build.

File: eo-runtime/src/test/java/org/eolang/PhiTest.java [hunk 2]
```diff
@@ -42,12 +42,15 @@ public void takesPackage() {
...
-                                Phi.Φ.attr("org").get(),
-                                "eolang"
+                                new PhMethod(
+                                    Phi.Φ.attr("org").get(),
+                                    "eolang"
+                                ),
+                            "math"
                             ),
-                        "math"
+                            "random"
```
Comment (yegor256) [inline eo-runtime/src/test/java/org/eolang/PhiTest.java:51]: @mximp why 8 indents?

File: eo-runtime/src/main/java/EOorg/EOeolang/EOmath/EOrandom$EOpseudo_seed.java [hunk 3]
```diff
@@ -24,28 +24,24 @@
...
+public class EOrandom$EOpseudo_seed extends PhDefault {
 
-    private static final Random RND = new SecureRandom();
-
-    public EOrandom(final Phi sigma) {
+    public EOrandom$EOpseudo_seed(final Phi sigma) {
         super(sigma);
         this.add("φ", new AtComposite(this, self -> new Data.ToPhi(
-            EOrandom.RND.nextDouble()
+                System.currentTimeMillis()
```
Comment (yegor256) [inline eo-runtime/src/main/java/EOorg/EOeolang/EOmath/EOrandom$EOpseudo_seed.java:44]: @mximp indentation broken

File: eo-runtime/src/main/eo/org/eolang/math/random.eo [hunk 4]
```diff
@@ -23,6 +23,48 @@
...
+        number 1
+        48
+      1
+
+  # Next random
+  random > next
+    $.next-seed
+
+  # New random
+  random > pseudo
```
Comment (yegor256) [inline eo-runtime/src/main/eo/org/eolang/math/random.eo:66]: @mximp it's better to keep the "surface" of an object as small as possible. Every "extra" method we introduce is a potential noise for object users. I suggest we stay with only two methods in this object: `next` and `pseudo`

**Comment by yegor256** [comment]
@mximp see above

**Comment by yegor256** [comment]
@mximp also, don't forget to fix the paper in `paper/` directory

File: eo-runtime/src/test/java/org/eolang/PhiTest.java (hunk 2 above)
Comment (mximp) [reply eo-runtime/src/test/java/org/eolang/PhiTest.java:51]: @yegor256 didn't get that - parameters are indented one under another.
Please clarify.

**Comment by mximp** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@mximp how about this test:

```
[] > seeded-randoms-are-equal
  assert-that > @
    (random 42).next
    $.equal-to
      (random 42).next
```

**Pull request merged**